AnkiDroid/src/main/java/com/ichi2/anki/pages/Statistics.kt

```diff
@@ -34,10 +36,18 @@ class Statistics : PageFragment() {
         get() = resources.getString(R.string.statistics)
 
     override val pageName = "graphs"
+    override fun onCreateView(
+        inflater: LayoutInflater,
+        container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View? {
+        return inflater.inflate(R.layout.statistics, container, false)
+    }
 
     override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
         super.onViewCreated(view, savedInstanceState)
-        view.findViewById<MaterialToolbar>(R.id.toolbar)?.apply {
+        val toolbar = view.findViewById(R.id.toolbar) as MaterialToolbar
+        toolbar.apply {
```

Comment:
There's a weak IDE warning here, in future see if these can be resolved (as they make the code easier to maintain)

```suggestion
        view.findViewById<MaterialToolbar>(R.id.toolbar).apply {
```